backend/src/test/java/de/bund/digitalservice/ris/norms/domain/entity/MetaTest.java

```diff
@@ -422,4 +418,53 @@ void returnsProprietaryEvenIfDoesNotExistInNorm() {
     // Then
     assertThat(result).isInstanceOf(Proprietary.class);
   }
+
+  @Test
+  void returnsOptionalProprietaryIfItExists() {
+    final Meta meta = Meta
+      .builder()
+      .node(
+        XmlMapper.toNode(
+          """
+          <akn:meta xmlns:akn="http://Inhaltsdaten.LegalDocML.de/1.7/" eId="meta-1" GUID="82a65581-0ea7-4525-9190-35ff86c977af">
+            <akn:proprietary eId="meta-1_proprietary-1" GUID="952262d3-de92-4c1d-a06d-95aa94f5f21c" source="attributsemantik-noch-undefiniert">
+              <meta:legalDocML.de_metadaten xmlns:meta="http://Metadaten.LegalDocML.de/1.7/">
+              <meta:typ>gesetz</meta:typ>
+              <meta:fna>754-28-1</meta:fna>
+              <meta:fassung>verkuendungsfassung</meta:fassung>
+              </meta:legalDocML.de_metadaten>
+            </akn:proprietary>
+          </akn:meta>
+          """
+        )
+      )
+      .build();
+
+    assertThat(meta.getProprietary()).isNotEmpty();
+  }
+
+  @Test
+  void returnsEmptyOptionalIfProprietaryDoesNotExist() {
+    // Given
+    var normXml =
+      """
+      <?xml-model href="../../../Grammatiken/legalDocML.de.sch" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+      <akn:akomaNtoso xmlns:akn="http://Inhaltsdaten.LegalDocML.de/1.7/"
+                      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                      xsi:schemaLocation="http://Metadaten.LegalDocML.de/1.7/ ../../../Grammatiken/legalDocML.de-metadaten.xsd">
+        <akn:act name="regelungstext">
+          <!-- Metadaten -->
+          <akn:meta eId="meta-1" GUID="000"></akn:meta>
+        </akn:act>
+      </akn:akomaNtoso>
```

Comment:
i would in general prefer to use .xml files loaded using `NormFixtures.loadNorm()` to create the norms in tests. That way a possible future conversion of the norms to a newer ldml version is a lot simpler.